telegram-bot-bumblebee-core/src/main/resources/application.yml

```diff
@@ -56,3 +56,10 @@ spring:
     username:
     password:
     driver-class-name:
+
+autocomplete:
+    template:
+      - ДО СВИ ДОС
+      - ДОС ВИ ДОС
```

Comment:
I will. And I think implementation of language-aware syllabus parsing for this small feature is a massive overkill.